packages/styles/variables.css

```diff
@@ -152,4 +152,9 @@
 
   /* list */
   --list-separator: rgba(153, 153, 153, 0.15);
+
+  /* tags */
+  --tag-text-color: #333333;
```

Comment:
we still haven't implemented our full color palette so for now, I'm adding new vars for tags...once we implement our palette I can replace references to these with the palette var names

Reply:
Is there an issue for adding the remaining variables?

Reply:
yes I believe so...there's a draft PR too